Meadowlark-js/tests/e2e/scenarios/QueryStringValidation.test.ts

```diff
@@ -34,6 +34,30 @@ describe('When retrieving information', () => {
       schools = bulkCreation.resources;
     });
 
+    describe('when querying changing the casing of the query value to be all lowercase', () => {
+      it('should return correct result', async () => {
+        await baseURLRequest()
+          .get(`/v3.3b/ed-fi/schools?nameOfInstitution=new school 1`)
+          .auth(await getAccessToken('host'), { type: 'bearer' })
+          .expect(200)
+          .then((response) => {
+            expect(response.headers['total-count']).toEqual('1');
+          });
+      });
+    });
+
+    describe('when querying changing the casing of the query value to be all uppercase', () => {
+      it('should return correct result', async () => {
+        await baseURLRequest()
+          .get(`/v3.3b/ed-fi/schools?nameOfInstitution=NEW SCHOOL 1`)
```

Comment:
@bradbanister doesn't this cover the casing test? "new school 1" above and "NEW SCHOOL 1" here.

Reply:
I was thinking in the actual `Meadowlark-js/backends/meadowlark-elasticsearch-backend/test/QueryElasticsearch.test.ts` tests. Something like `describe('when querying with a case different from the case in the datastore')`

Reply:
That explains quite clearly that case-independent search is supported

Reply:
Also, to be blunt the e2e tests are not very helpful to me during development. Half the time I can't get them to run locally without container configuration errors that need lots of manual intervention.